src/components/SetsForm.jsx

```diff
@@ -0,0 +1,77 @@
+import {nanoid} from 'nanoid';
+import {useRouter} from 'next/router';
+import {useState} from 'react';
+
+import useStore from '../hooks/useStore';
+
+import MinusIcon from './Icons/MinusIcon';
+import PlusIcon from './Icons/PlusIcon';
+import RepetitionIcon from './Icons/RepetitionIcon';
+import MoveBackButton from './MoveBackButton';
+import StyledButton from './StyledButtons';
+import StyledForm from './StyledForms';
+import {StyledHeader, StyledH1} from './StyledHeader';
+import StyledInput from './StyledInput';
+import SubmitButton from './SubmitButton';
+export default function SetsForm() {
+	const router = useRouter();
+	const [setsCounter, setSetsCounter] = useState(0);
+
+	function decrementRepetition() {
+		setSetsCounter(setsCounter - 1);
+	}
+
+	function incrementCounter() {
+		setSetsCounter(setsCounter + 1);
+	}
+	const addSetsCounter = useStore(state => state.addSetsCounter);
+
+	function handleSubmit(event) {
+		event.preventDefault();
+
+		const setsCounterObj = {
+			id: nanoid(),
+			setsCounter: setsCounter,
+		};
+		addSetsCounter(setsCounterObj);
+		router.push('/');
```

Comment:
For better readability I would suggest not to use abbreviations. Anyhow, I think you do not need to name it "object" cause from your code it is very clear that this is an object. So maybe another name would be more fitting, something like `newSetsCounter` for example.
```suggestion
		const setsCounterObject = {
			id: nanoid(),
			setsCounter: setsCounter,
		};
		addSetsCounter(setsCounterObject);
		router.push('/');
```

Reply:
fir me this naming convention is good to understand what is going on, since i add also an id to that object. So i would prefer to keep the name as object for my understanding while i read the code. 
But you are right i will not abbreviate the name, i will change it to "setsCounterObjective

Reply:
`setCounerObject` as it is an object 👍🏼